adapter.go

```diff
@@ -194,6 +196,7 @@ func NewAdapterByDBUseTableName(db *gorm.DB, prefix string, tableName string) (*
 	a := &Adapter{
 		tablePrefix: prefix,
 		tableName:   tableName,
+		customTable: db.Statement.Context.Value(customTableKey{}),
```

Comment:
Instead of using Context to pass customTable, you can use a.customTable directly